tutorcairn/patches/local-docker-compose-services

```diff
@@ -85,3 +85,17 @@ cairn-postgresql:
     depends_on:
         - permissions
 {% endif %}
+cairn-watchcourses:
+    image: {{ DOCKER_IMAGE_OPENEDX }}
+    command: >
+        bash -c "pip install -qr /openedx/scripts/requirements.txt
+        && uvicorn --app-dir /openedx/scripts/ main:app --host 0.0.0.0 --port {{ CAIRN_WATCHCOURSES_PORT }}"
+    restart: unless-stopped
+    environment:
+      SETTINGS: ${TUTOR_EDX_PLATFORM_SETTINGS:-tutor.production}
```

Comment:
Can we avoid defining this environment variable here? After all, the fastapi process only needs it for its subprocess, not its main process. I suggest removing it from the service declaration and use the `env` option in `subprocess.call(... env=...)` ([docs](https://docs.python.org/3/library/subprocess.html#popen-constructor)).

Reply:
It just works without defining this environment variable. Checking the value of `os.environ`, it reveals that the container is already using the production settings.

This is what I found in `os.environ`.
```
'DJANGO_SETTINGS_MODULE': 'lms.envs.tutor.production'
```